main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import
+	"fmt"
+	"os"
+
+"github.com/aws/aws-sdk-go/aws"
+"github.com/aws/aws-sdk-go/aws/session"
+"github.com/aws/aws-sdk-go/service/dynamodb"
+"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+
+
+// Define a struct to hold the item data
+type Item struct {
+	ID   int    `json:id`
+	Name string `json:name`
+}
+
+func main() {
+	// Create a session with AWS
+	sess, err = session.NewSession(&aws.Config{
+		Region: aws.String("australia"),
```

Comment:
Would this best be configurable ? env var, CLI param etc ? This would be useful for multiregion deployment.